### ISSUE TITLE
Support for react-native-reanimated v2

### ISSUE THREAD

**Issue by RatebSeirawan** [gorhom/react-native-paper-onboarding#26]
# Feature Request

Support for react-native-reanimated v2

## Why it is needed

from react-native-reanimated v2 migration docs it says that it's backward compatible with some changes like changing 
```
import {interpolate} from 'react-native-reanimated';
// to
import {interpolateNode} from 'react-native-reanimated';
```

I tried renaming all the imports but it won't work, I'm currently trying to re-implement the component with the new API but it needs some learning curve to do so.

**Comment by drewandre** [comment]
@RatebSeirawan did you get "malformed js calls" error (<NaN> cannot be used as animated value) after changing interpolate -> interpolateNode? That's what I was seeing.

**Comment by RatebSeirawan** [comment]
@drewandre  yes, js error like yours

**Comment by gorhom** [comment]
i have looked into the issue, seems it is not backward compatibility 100% , migrating to Reanimated V2 will need a refactor :\

**Comment by gorhom** [comment]
this should be resolved with v1.1.0 👍
